app/models/student.rb

```diff
@@ -0,0 +1,33 @@
+class Student < ActiveRecord::Base
+	belongs_to:standard
+	validates:first_name,presence:true
+	validates:middle_name,presence:true
+	validates:last_name,presence:true
+	validates:date_of_birth,presence:true
+	validates:address,presence:true
+	validates:city,presence:true
+	validates:contactno,presence:true,numericality:{only_integer:true}
+	validates:standard_id,presence:true
```

Comment:
You can set validations like
validates :first_name, :middle_name, :last_name, ...., presence: true